priors/sos/Sos.py

```diff
@@ -293,18 +293,30 @@ def _create_dims_vars(self, sos):
             sos NetCDF Dataset
         """
 
-        oi = sos["model"].createVariable("overwritten_indexes", "i4", ("num_reaches",))
-        oi.comment = "Indexes of GRADES priors that were overwritten."
+        try:
+            oi = sos["model"].createVariable("overwritten_indexes", "i4", ("num_reaches",))
+            oi.comment = "Indexes of GRADES priors that were overwritten."
+        except:
+            print("overwritten_indexes allready created")
+
+        try:    
+            sos["model"].createDimension("nchar", 4)
+            os = sos["model"].createVariable("overwritten_source", "S1", ("num_reaches", "nchar"))
+            os.comment = "Source of gage data that overwrote GRADES priors."
+        except:
+            print('overwritten_source already created')
```

Comment:
Could we also take out the print statements here and use a conditional if statement to check if the "overwritten_source" variable exists and then create it if needed? And the same for "bad_prior_source".